docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,90 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
+
+## What is it?
+
+Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+
+When using extendable bicep parameter files, you will have a main `.bicepparam` file that can be consumed by multiple extended `.bicepparam` files.
+
+## Using
+
+`main.bicep` This is your main bicep file, which will define your parameters for deployment.
+
+```bicep
+param namePrefix string
+param location string
+param symbolicName string
+```
+
+`root.bicepparam` This is your main bicepparam file, which can be reused by multiple extended .bicepparam files and in multiple deployments.
```

Comment:
Can you explicitly call out the usage of the `using none`, which is a new concept? Something like: "Notice that the first line of this bicepparam file declares `using none` which tells the compiler not to validate this against any particular `.bicep` file.